src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -354,6 +360,8 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
 
         const { accessStats, accessStatsError, file, fileError, isLoadingAccessStats }: State = this.state;
 
+        const shouldShowArchivedAt = isFeatureEnabled(features, 'details.archivedAt.enabled');
```

Comment:
I feel like this should be moved all the way to down the bottom where the `archivedAt` is actually used, if you put a tenary at this level to decide if it should have the `archivedAt` or not because you wont know if the `archivedAt` is actually present but the split is off or its undefined when its retrieved from the server or if the split is on and the `archivedAt` is undefined

Reply:
Seems better. Just to make sure - you mean move features all the way down to `ItemProperties.js` or should I just move the `shouldShowArchivedAt` as a prop?